README.md

```diff
@@ -110,10 +112,18 @@ from domain.apis import Foo
 
 This keeps namespaces tidy and does not leak domain details.
 
-* A domain **does not** need to have all these files if it is not using them. For example - a domain that just co-ordinates API calls to other domains does not need to have `models.py` as it is probably not storing anything in a datastore.
+* A domain **does not** need to have all these files if it is not using them. For example - a domain that just coordinates API calls to other domains does not need to have `models.py` as it is probably not storing anything in a datastore.
 
 * A domain **can have** additional files when it makes sense (such as `utils.py` or `enums.py` or `serializers.py`) to separate out parts of the code that aren't covered by the styleguide pattern.
 
+## Which logic lives where?
+
+It's common in programming to end up confused about what type of logic should live where - should it go in the `apis.py`, the `models.py`, or the `services.py`? There are many cases where it's difficult to decide, and the best advice is to **pick a pattern and stick to it**, but for simpler things, this guide emphasises the following:
+
+- apis.py - logic about presentation (Where should I show this data to the user? Where do I define the API schema?)
+- services.py - logic around coordination and transactions (Where do I coordinate updating many models in one domain? Where do I dispatch a single action out to other domains?)
+- models.py - logic around information (Where can I store this data? Where can I do any post/pre-save actions?)
```

Comment:
could also add, "derived information computed by already existing information" (e.g. the discounted order total example)